scripts/build.js

```diff
@@ -0,0 +1,173 @@
+'use strict';
+
+const utility = require('./utility');
+const ts = require('typescript');
+const externals = require('./rollup-externals');
+
+const currentPath = process.argv[1].slice(0, process.argv[1].lastIndexOf('/'));
+const tscES5OutDir = `/lib`;
+const tscES6OutDir = `/lib-esm`;
+const packageInfo = require(`${currentPath}/package`);
+
+async function buildRollUp() {
+	console.log(`Building Roll up bundle file under ${currentPath}`);
+	const rollup = require('rollup');
+	const resolve = require('rollup-plugin-node-resolve');
+	const sourceMaps = require('rollup-plugin-sourcemaps');
+	const json = require('rollup-plugin-json');
+
+	const input = `${currentPath}${tscES5OutDir}/index.js`;
+	const file = `${currentPath}${packageInfo.main.slice(
+		packageInfo.main.indexOf('/')
+	)}`;
+	const onwarn = warning => {
```

Comment:
Not sure if this is the reason for this `onwarn` trick, if so, it would be nice to have the context

```suggestion
+ 	// For more info see: https://github.com/rollup/rollup/issues/1518#issuecomment-321875784
	const onwarn = warning => {
````